packages/driver/cypress/integration/commands/actions/scroll_spec.js

```diff
@@ -396,153 +459,174 @@ describe('src/cy/commands/actions/scroll', () => {
       })
 
       it('waits until the subject is scrollable', () => {
-        cy.stub(cy, 'ensureScrollability')
-        .onFirstCall().throws(new Error())
+        cy.stub(cy, 'ensureScrollability').onFirstCall().throws(new Error())
 
         cy.on('command:retry', () => {
           return cy.ensureScrollability.returns()
         })
 
-        cy
-        .get('#scroll-into-view-horizontal')
-        .scrollTo('right').then(() => {
+        cy.get('#scroll-into-view-horizontal')
+        .scrollTo('right')
+        .then(() => {
           expect(cy.ensureScrollability).to.be.calledTwice
         })
       })
     })
 
-    describe('errors', {
-      defaultCommandTimeout: 50,
-    }, () => {
-      beforeEach(function () {
-        this.logs = []
-
-        cy.on('log:added', (attrs, log) => {
-          this.lastLog = log
-          this.logs.push(log)
-        })
-
-        return null
-      })
+    describe(
+      'errors',
+      {
+        defaultCommandTimeout: 50,
+      },
+      () => {
```

Comment:
another example of something not matching how we prescribe to users to do this. we'd suggest something more compact like:

```suggestion
    describe('errors', { defaultCommandTimeout: 50 }, () => {
```

Reply:
yeah. i wonder if there could be a prettier-cypress-plugin someday. IMO, in tests, it makes sense to have some specialized linting, since the `describe(...it(...` stuff is almost like a DSL

Reply:
Thanks @flotwig this is precisely why I was saying this is a full on product change, and something we need to be very prescriptive and careful about. The `describe` and `it` is a fluid DSL that's not superficial code - it's **the format** and outline for the contract between value and our users.